### PR TITLE
Fix travis

### PR DESCRIPTION
**Pull request by AnshulMalik**
I removed some redundant code, we can add it later when actually required.

### CODE REVIEW

**Comment by nileshgulia1** [comment]
Thanks @AnshulMalik  for changes, assuming that `react-split-pane` not needed right now, may be @mbebenita  can review it ,Otherwise good work :)

File: .travis.yml [hunk 1]
```diff
@@ -13,12 +13,7 @@ notifications:
   email:
     on_success: never
     on_failure: always
-before_install:
-- npm update
 install:
 - npm install
-before_script:
-- npm run clean
```
Comment (nileshgulia1) [inline .travis.yml]: May be there is no need of removing clean script ,we can add this to `package.json`:
`"clean": "rm -r dist/*"` what it does is just removes a bunch of files so you can start with a fresh working copy to start building into.

**Comment by mbebenita** [comment]
> assuming that react-split-pane not needed right now

You are correct that `react-split-pane` is not needed and can be removed.

**Comment by mbebenita** [comment]
@AnshulMalik when submitting PRs, please don't include merge commits because it makes it difficulty to review code. Usually, you'll want to rely on git rebase to have all your commits in a linear history rebased on the current master.

**Comment by mbebenita** [comment]
Looks good.

**Pull request merged**